### ISSUE TITLE
New survey for Teachers

### ISSUE THREAD

**Issue by unintendedbear** [unintendedbear/IEEE-Transactions-on-Education#1]
And, should we make another for parents, @mgarenas?

**Comment by mgarenas** [comment]
No, we should do exactly in the same way than we do previously for reusing the data

Enviado desde mi dispositivo Samsung

-------- Mensaje original --------
De: Paloma de las Cuevas Delgado notifications@github.com 
Fecha: 03/10/2016  11:05  (GMT+01:00) 
Para: unintendedbear/IEEE-Transactions-on-Education IEEE-Transactions-on-Education@noreply.github.com 
Cc: Maribel García Arenas mgarenas@ugr.es, Mention mention@noreply.github.com 
Asunto: [unintendedbear/IEEE-Transactions-on-Education] New survey for
  Teachers (#1) 

And, should we make another for parents, @mgarenas?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/unintendedbear/IEEE-Transactions-on-Education","title":"unintendedbear/IEEE-Transactions-on-Education","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/unintendedbear/IEEE-Transactions-on-Education"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"New survey for Teachers (#1)"}],"action":{"name":"View Issue","url":"https://github.com/unintendedbear/IEEE-Transactions-on-Education/issues/1"}}}

**Comment by unintendedbear** [comment]
The fact that we make new surveys for teachers and/or parents does not mean the we cannot use previous data for students if we do not change students' surveys, they are different things.

**Comment by mgarenas** [comment]
Ok, but what do you want ask to parents? Are these questions important for the results? Are there any papers relevant for parents' opinion? 
Althought I think the main problem is how many parents will be able for replying the survey? I know, we are not able to know previously.
A
 

Enviado desde mi dispositivo Samsung

-------- Mensaje original --------
De: Paloma de las Cuevas Delgado notifications@github.com 
Fecha: 03/10/2016  14:06  (GMT+01:00) 
Para: unintendedbear/IEEE-Transactions-on-Education IEEE-Transactions-on-Education@noreply.github.com 
Cc: Maribel García Arenas mgarenas@ugr.es, Mention mention@noreply.github.com 
Asunto: Re: [unintendedbear/IEEE-Transactions-on-Education] New survey for
  Teachers (#1) 

The fact that we make new surveys for teachers and/or parents does not mean the we cannot use previous data for students if we do not change students' surveys, they are different things.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/unintendedbear/IEEE-Transactions-on-Education","title":"unintendedbear/IEEE-Transactions-on-Education","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/unintendedbear/IEEE-Transactions-on-Education"}},"updates":{"snippets":[{"icon":"PERSON","message":"@unintendedbear in #1: The fact that we make new surveys for teachers and/or parents does not mean the we cannot use previous data for students if we do not change students' surveys, they are different things."}],"action":{"name":"View Issue","url":"https://github.com/unintendedbear/IEEE-Transactions-on-Education/issues/1#issuecomment-251089957"}}}

**Comment by unintendedbear** [comment]
That's the problem, I did not see much about the parents when obviously are part of the problem.
But ok, I will include only a survey for teachers, for which I do have a chance to obtain results.
